web/modules/custom/dpl_patron_reg/src/Controller/DplPatronRegController.php

```diff
@@ -145,6 +149,11 @@ public function authRedirect(Request $request, string $client_name): TrustedRedi
     $url->setAbsolute();
     $url = $url->toString();
 
+    // Set special cookie to detect this is patron registration flow. This is
+    // used in dpl_login.
+    $store = $this->privateTempStore->get('dpl_patron_reg');
+    $store->set('isRegistrationFlow', TRUE);
```

Comment:
Suggestion: Instead of reading/deleting from the tempstore in an imperative way, it would be nice with some methods in the dpl_patron_reg namespace: Eg.: Something::userIsInRegistrationFlow() and Something::clearUserRegistrationFlowStatus() and Something::setUserIsInRegistrationFlowStatus().